tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2190,11 +2198,16 @@ public function userRestoresSpaceObjectsFromTrashRequest(
 
 		// find object in trash
 		$objectsInTrash = $this->getObjectsInTrashbin($user, $spaceName);
+		$pathToDeletedObject = "";
```

Comment:
The IDE mentioned that `$pathToDeletedObject` might not be defined. So define it.
I throw an exception below, if the required object can't be found at all - it seems better to fail in a "cpntrolled" way if that happens, rather than let the code run on, and fail somewhere that will be harder to understand.